diff --git a/node_modules/@segment/analytics-next/dist/pkg/lib/load-script.js b/node_modules/@segment/analytics-next/dist/pkg/lib/load-script.js
index c3db17c..3ca753a 100644
--- a/node_modules/@segment/analytics-next/dist/pkg/lib/load-script.js
+++ b/node_modules/@segment/analytics-next/dist/pkg/lib/load-script.js
@@ -20,6 +20,8 @@ export function loadScript(src, attributes) {
         var _a;
         var script = window.document.createElement('script');
         script.type = 'text/javascript';
+        // Allow segment scripts to work in environments that use strict Cross-Origin-Embedder-Policy policies.
+        script.crossOrigin = 'anonymous';
         script.src = src;
         script.async = true;
         script.setAttribute('status', 'loading');
