diff --git a/node_modules/@winglang/sdk/lib/shared-aws/function.js b/node_modules/@winglang/sdk/lib/shared-aws/function.js
index 430bf1b..378a899 100644
--- a/node_modules/@winglang/sdk/lib/shared-aws/function.js
+++ b/node_modules/@winglang/sdk/lib/shared-aws/function.js
@@ -3,7 +3,7 @@ var _a;
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.Function = void 0;
 const JSII_RTTI_SYMBOL_1 = Symbol.for("jsii.rtti");
-const target_tf_aws_1 = require("../target-tf-aws");
+
 /**
  * A helper class for working with AWS functions.
  */
